src/content/app/genome-browser/components/browser-location-indicator/BrowserLocationIndicator.tsx

```diff
@@ -20,21 +20,23 @@ import classNames from 'classnames';
 
 import { getCommaSeparatedNumber } from 'src/shared/helpers/formatters/numberFormatter';
 
+import { useGenomeKaryotypeQuery } from 'src/shared/state/genome/genomeApiSlice';
+
 import { getBrowserActiveGenomeId } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSelectors';
 import { getActualChrLocation } from 'src/content/app/genome-browser/state/browser-general/browserGeneralSelectors';
 import { toggleBrowserNav } from 'src/content/app/genome-browser/state/browser-nav/browserNavSlice';
-import { getGenomeKaryotype } from 'src/shared/state/genome/genomeSelectors';
 
 import styles from './BrowserLocationIndicator.scss';
 
-type Props = {
+export type Props = {
```

Comment:
Since we are exporting it, can it be changed to `BrowserLocationIndicatorProps`?

Reply:
✅